docs/porting-guides/6.x.x.md

```diff
@@ -111,6 +111,15 @@ The following data model keys have been removed from `eos_designs` in AVD 6.0.0.
 | `underlay_mutlticast` | `underlay_mutlticast_pim_sm` or `<node_type_key>.defaults.underlay_multicast.pim_sm.enabled` or `<node_type_key>.node_groups[].underlay_multicast.pim_sm.enabled` or `<node_type_key>.node_groups[].nodes[].underlay_multicast.pim_sm.enabled` or `<node_type_key>.nodes[].underlay_multicast.pim_sm.enabled` |
 | `underlay_ospf_authentication.message_digest_keys[].key` | `underlay_ospf_authentication.message_digest_keys[].cleartext_key` |
 
+### Enhanced conflict detection in `eos_designs`
+
+In AVD 5.x, conflict detection was done per data model (e.g., `network_services`, `connected_endpoints`, ...).
+This implies that it was possible to declare conflicting information between data models (e.g., overwriting an MLAG interface in `connected_endpoints`), which could lead to difficult-to-track errors.
```

Comment:
```suggestion
This implies that it was possible to declare conflicting information between data models (e.g. overwriting an MLAG interface in `connected_endpoints`), which could lead to difficult-to-track errors.
```

Reply:
https://www.scribbr.com/definitions/for-example-abbreviation/#:~:text=The%20latin%20abbreviation%20for%20%E2%80%9Cfor%20example%E2%80%9D%20is%20e.g.%2C%20which,)%20are%20native%20to%20Australia.%E2%80%9D